tests/core/gadget-config-defaults-vitality/gadget-vitality-hint.yaml

```diff
@@ -0,0 +1,4 @@
+defaults:
+   system:
+     resilience:
+       vitality-hint: "unrelated-snap,test-snapd-with-configure,test-snapd-with-configure-core18"
```

Comment:
as I mentioned in a previous comment I think we want the gadget itself in this list too and to have a service in it (added in the test when we repack it)